### DISCUSSION TITLE
Ada Lovelace FP8 MM support

### DISCUSSION THREAD

**Discussion by alecco** [NVIDIA/cutlass#1343] [Ideas]
Are there plans to add support for Ada Lovelace cards? In particular FP8 tensor.

Thanks.

**Comment by hwu36** [comment]
we will support it when nvcc supports it.  it is coming.

**Comment by alecco** [comment]
FWIW this seems to be supported in latest CUTLASS 3.5 (need CUDA 12.4+).
